README.md

```diff
@@ -0,0 +1,60 @@
+# KNOT Service for Linux systems
+
+KNOT service is part of KNOT project. It aims to provide a "proxy" service
+for KNOT nodes, allowing power constrained embedded device to interact
+with Meshblu cloud services.
+
+The initial target platform are nRF24L01 nodes, and Raspiberry PI GW. nRF24L01
```

Comment:
Update this text.
You can use these snippets removed from the KNoT documentation:

> KNoT Devices Manager (knotd). The knotd is a daemon acting as proxy between the network managers and the KNoTFog, thus providing a central point for handling KNoT devices, protocol translation, and message forwarding capabilities.

> The inetbrd daemon creates an IP bridge for sending the traffic from local interfaces to the KNoT Device Manager (knotd).

Reply:
Also, you can add any information about supporting the nRF52840.